pushover.sh

```diff
@@ -33,6 +33,8 @@ showHelp()
         echo "                                0 - normal priority"
         echo "                                1 - bypass the user's quiet hours"
         echo "                                2 - require confirmation from the user"
+        echo "  -E, --expire               Set expiration time for notifications with priority 2 (default 180)"
```

Comment:
Why use a capital E and R instead of just lowercase e and r, this would keep them in line with the other options

Reply:
I used uppercase to mark these non-required optional parameters with default values.
Can use lowercase instead.

Reply:
Is it a typical standard for CLI tools to make optional parameters uppercase?

Reply:
I tried to find some documentation on CLI options and best practices but only found this http://docopt.org/
I'd appreciate it if you could make those lowercase and I'll get this approved and merged, and thanks for the addition!

Reply:
I would also add 2 readonly variables at the top for EXPIRE_DEFAULT and RETRY_DEFAULT, then use those in place of your hard 180 and 30 both in the usage an in the check further down in the script

Reply:
```suggestion
        echo "  -e, --expire               Set expiration time, in seconds, for notifications with priority 2 (default ${EXPIRE_DEFAULT})"
```